package.json

```diff
@@ -151,6 +151,7 @@
     "stylelint-config-recommended": "^3.0.0",
     "svg-to-jsx": "^1.0.2",
     "tinycolor2": "^1.4.1",
+    "ts-migrate": "^0.1.26",
     "typescript": "4.2.2",
```

Comment:
Just noticed this: There are some really helpful improvements in `typescript@4.3.0`, might be good to update to latest soon.

Reply:
Sweet. I'll follow up with a MR to update that and a bunch of other packages.